sonar-scanner-engine/src/main/java/org/sonar/scanner/scan/filesystem/FileIndexer.java

```diff
@@ -103,7 +103,7 @@ void index(DefaultModuleFileSystem fileSystem) {
     progressReport.stop(progress.count() + " files indexed");
 
     if (exclusionFilters.hasPattern()) {
-      LOG.info(progress.excludedByPatternsCount() + " files ignored because of inclusion/exclusion patterns");
+      LOG.info("{} files ignored because of inclusion/exclusion patterns", progress.excludedByPatternsCount());
```

Comment:
to be perfect, the call to excludedByPatternsCount() can be avoided if info level is disabled by using a Supplier.